packages/web3-react/src/hooks/useAutoConnect.ts

```diff
@@ -48,16 +48,20 @@ export const useEagerConnector = (connectors: ConnectorsContextValue) => {
       })();
       if (!connector) return;
 
-      if (shouldAutoConnectApp) {
-        const onContinue = () => {
-          activate(connector, undefined, true);
-        };
-        acceptTermsModal.setOnContinue?.(() => onContinue);
+      const connectWallet = () => activate(connector, undefined, true);
+
+      let termsAccepted = false;
+      if (typeof window !== 'undefined') {
+        termsAccepted = window.localStorage?.getItem(LS_KEY_TERMS_ACCEPTANCE) === 'true';
+      }
```

Comment:
you can use useLocalStorage instead

Reply:
@Jabher 
yes, but I don't actually need it here